android/build.gradle

```diff
@@ -1,15 +1,17 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
     repositories {
+        google()
```

Comment:
Is this necessary :) ?

Reply:
`google()` is a now the formal repository for Android related libraries, like plugins, support libs, etc. ([here](https://dl.google.com/dl/android/maven2/index.html) are the contents).